docs/normalize-intervals.rst

```diff
@@ -0,0 +1,827 @@
+.. _normalize-intervals:
+
+====================================
+Normalize time series data intervals
+====================================
+
+.. sidebar:: The ISS Orbit
+
+    .. image:: _assets/img/normalize-intervals/orbit.png
+        :alt: An animated visualization of the ISS orbit
+        :target: https://en.wikipedia.org/wiki/International_Space_Station#/media/File:Animation_of_International_Space_Station_trajectory.gif
+
+    The ISS travels at 27,724 kilometers per hour and orbits Earth
+    approximately once every 90 minutes. Each complete wave on the graph
+    represents a full orbit. The amplitude oscillates
+
+If you followed one of the tutorials in the :ref:`previous section <gen-ts>`,
+you should have some mock time-series data about the position, or `ground
```

Comment:
for consistency:

"time series" instead of "time-series"

Reply:
ah, I have been pondering this. I figured this was a good approach:

- "time series" when used a phrasal noun (e.g., "this is a time series")
- "time-series" when used as a phrasal adjective before a noun (e.g., "time-series data")

cf. https://grammarist.com/grammar/phrasal-adjectives/

@matthijskrul what are your thoughts on this? I am open to correction, suggestions, etc

Reply:
I will spin this off into a separate discussion. changing to "time series" for now